src/task_queue/database_tasks/backfill_data.py

```diff
@@ -0,0 +1,343 @@
+                community_member: CommunityMember = CommunityMember.objects.filter(
+                    community=community, user=user
+                ).first()
+                if community_member:
+                    community_member.is_admin = False
+                    community_member.save()
+                else:
+                    community_member = CommunityMember.objects.create(
+                        community=community, user=user, is_admin=False
+                    )
+        admin_groups = CommunityAdminGroup.objects.all()
+        for group in admin_groups:
+            for member in group.members.all():
+                community_member: CommunityMember = CommunityMember.objects.filter(
+                    community=group.community, user=member
+                ).first()
+                if community_member:
+                    community_member.is_admin = True
+                    community_member.save()
+                else:
+                    community_member = CommunityMember.objects.create(
+                        community=group.community, user=member, is_admin=True
+                    )
+        return data
+    except Exception as e:
+        capture_message(str(e), level="error")
+        data.append({"Message": "Fatal error", "Error": str(e)})
+        return data
+
+
+def backfill_graph_default_data():
+    data = []
+    try:
+        for community in Community.objects.all():
+            for tag in TagCollection.objects.get(
+                name__icontains="Category"
+            ).tag_set.all():
+                d = Data.objects.filter(community=community, name=tag.name).first()
+                if d:
+                    oldval = d.value
+                    val = 0
+                    if community.is_geographically_focused:
+                        user_actions = UserActionRel.objects.filter(
+                            real_estate_unit__community=community, status="DONE"
+                        )
+                    else:
+                        user_actions = UserActionRel.objects.filter(
+                            action__community=community, status="DONE"
+                        )
+                    for user_action in user_actions:
+                        if (
+                            user_action.action
+                            and user_action.action.tags.filter(pk=tag.id).exists()
+                        ):
+                            val += 1
+                    d.value = val
+                    d.save()
+                    message = "Community: %s, Category: %s, Old: %s, New: %x" % (
+                        community.name, tag.name, str(oldval), str(val))                    
+                    data.append({"Message": message, "Error":""})
+
+        return data
+    except Exception as e:
+        capture_message(str(e), level="error")
+        data.append({"Message": "Fatal error", "Error": str(e)})
+        return data
+
+
+def backfill_locations():
```

Comment:
i dont know if this is old code or not but this is a bit much.
At best we need to break this down into smaller functions.

In fact, maybe we should have a separate file with this as a script with several smaller functions and then we can call the main part of that script here.  this is a bit hard to review